recipes/hapnest/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "hapnest" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/intervene-EU-H2020/synthetic_data/archive/refs/heads/main.zip
+  sha256: 6457c711e3430420d07b2ca8e22303a08ab60811b34c9f005a4c87bbe1ea1ade
+
+build:
+  number: 0
+  skip: true
```

Comment:
I think this is why you aren't getting a build.

Reply:
Thank you @aliciaaevans, skip line deleted.